react/week1/todos/src/index.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+// import App from './App';
+// import * as serviceWorker from './serviceWorker';
+
+
+const root = document.getElementById('root')
+
+ function TodoItem(props) {
+   return <li>{props.description}, {props.deadline}</li>
+ }
+
+class Todo extends React.Component {
+  render() {
+   const todoList = this.props.todos;
+   const result = todoList.map(element => {
+    return <TodoItem description={element.description} deadline={element.deadline}></TodoItem>
```

Comment:
👍

Reply:
Thanks for the feedback @fernandoabolafio I have made necessary changes.